server/game/cards/11-DoR/TheStrengthOfTheMountain.js

```diff
@@ -0,0 +1,27 @@
+const DrawCard = require('../../drawcard.js');
+
+class TheStrengthOfTheMountain extends DrawCard {
+    setupCardAbilities(ability) {
```

Comment:
From my understanding the `ability` parameter has been deprecated.  Include AbilityDsl (`const AbilityDsl = require('../../abilitydsl');`) and use `AbilityDsl.` instead of `ability`.